contents/home.mdx

```diff
@@ -1,5 +1,25 @@
 # science, code, and open source
 
+## Sciwork Conference 2024
+Welcome to the **Sciwork Conference 2024**! Join us for an exciting day of training, and collaboration opportunities with experts and enthusiasts from the tech community.
+
+## NASA TOPS Training
+
+Please refere <a href="/training">Training</a>
```

Comment:
There is a typo need to be fixed with the change `Please refer to the <a href="/training">Training</a> page.`

Reply:
Thanks for pointing out this typo !